VotingApplication/VotingApplication.Web/App_Start/BundleConfig.cs

```diff
@@ -83,6 +83,19 @@ public static void RegisterBundles(BundleCollection bundles)
             errorBarStyle.Orderer = nullOrderer;
             bundles.Add(errorBarStyle);
 
+            StyleBundle voteOnStyle = new StyleBundle("~/Bundles/VoteOnStyle");
+            voteOnStyle.Include("~/Content/Scss/VoteOn.scss");
+            voteOnStyle.Builder = nullBuilder;
+            voteOnStyle.Transforms.Add(styleTransformer);
+            voteOnStyle.Orderer = nullOrderer;
+            bundles.Add(voteOnStyle);
+
+            StyleBundle angularMaterialExtensions = new StyleBundle("~/Bundles/AngularMaterialExtensions");
```

Comment:
Similar to the above, prefixing the bundle with "Angular" implies it's from google, rather than a different source.